docs/src/pages/components/tree-view/GmailTreeView.tsx

```diff
@@ -29,57 +30,38 @@ type StyledTreeItemProps = TreeItemProps & {
   labelText: string;
 };
 
-const useTreeItemStyles = makeStyles((theme: Theme) => ({
-  root: {
-    color: theme.palette.text.secondary,
-  },
-  content: {
+const StyledTreeItemRoot = experimentalStyled(TreeItem)(({ theme }) => ({
+  color: theme.palette.text.secondary,
+  [`& .${treeItemClasses.content}`]: {
```

Comment:
Interesting, it's seems that we didn't use them in the past. What should we do going forward?

Reply:
My thinking was, if there are more styles, especially with multiple keys to style a component, `styled()` looked like the better replacement (I just replaced the keys in the classes with the classes selectors). For trivial, one line changes I used the `sx` prop.

Reply:
I like the concept of being able to trust already defined class names instead of typing and making a mistake on it.

It looks interesting to use the same technique that is already used in the creation of the component.

Reply:
My point was exactly on what @vicasas, the use of the classes object. No really preferences. I like how a plain string is YOLO and often good enough, but I also understand how having a classes. with intellisense that describe what each key is for can also help.